packages/block-editor/src/components/link-control/style.scss

```diff
@@ -184,13 +184,14 @@ $preview-image-height: 140px;
 		position: relative;
 		top: 0.2em;
 		margin-right: $grid-unit-10;
-		width: 18px; // half of 32px to improve perceived resolution.
-		height: 18px; // half of 32px to improve perceived resolution.
+		max-height: 24px;
+		flex-shrink: 0;
+		width: 24px;
+		display: flex;
+		justify-content: center;
```

Comment:
Nit: to make things easier for future devs and to avoid regressions it might be nice to add a code comment here to explain why we're doing this and what issue it addresses? Otherwise I'm concerned this would be open to accidental regression.